docs/reference/argocd.md

```diff
@@ -156,6 +156,42 @@ spec:
     resources: {}
 ```
 
+The following example shows how to set command line parameters using the env variable 
+
+``` yaml
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: example-argocd
+  labels:
+    example: controller
+spec:
+  controller:
+    env:
+    - name: ARGOCD_APPLICATION_CONTROLLER_REPO_SERVER_TIMEOUT_SECONDS
+      value: '120'    
+```
+
+[Refer for more details] (https://argo-cd.readthedocs.io/en/stable/operator-manual/server-commands/additional-configuration-method/) 
+
+If you want to use the config map (as mentioned in the link above) to set the cmd variables, you can add this to the CR, to refer to the value
+
+``` yaml
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: example-argocd
+  labels:
+    example: controller
+spec:
+  controller:
+    env:
+    - name: ARGOCD_APPLICATION_CONTROLLER_REPO_SERVER_TIMEOUT_SECONDS
+      valueFrom:
+      configMapKeyRef:
+        key: controller.repo.server.timeout.seconds
+        name: argocd-cmd-params-cm     
```

Comment:
Tiny nit: I think this should be intended one level deeper.

Reply:
You are correct, will update it